File: KMPSampleIOSApp/KMPSampleIOSAppTests/utils/PriorityQueueUtilTest.swift
@testable import KMPSampleIOSApp
import Testing

struct PriorityQueueUtilTest {
    @Test func pick() async throws {
        #expect(PriorityQueueUtil.pickMinimum(from: [1, 2, 5, 7, 10]) == 1)
    }
}

File: KMPSampleAndroidApp/app/src/test/java/dev/kaseken/kmpsampleandroidapp/utils/PriorityQueueUtilTest.kt
package dev.kaseken.kmpsampleandroidapp.utils

import org.junit.Assert
import org.junit.Test

class PriorityQueueUtilTest {
  @Test
  fun testPickMinimum() {
    Assert.assertEquals(1, PriorityQueueUtil.pickMinimum(listOf(1, 2, 5, 7, 10)))
  }
}
